test/language/statements/class/elements/private-getter-brand-check-multiple-evaluations-of-class-realm.js

```diff
@@ -0,0 +1,54 @@
+// Copyright (C) 2019 Jaideep Bhoosreddy (Bloomberg LP). All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+description: Every new evaluation of a class creates a different brand (private getter)
+esid: sec-privatefieldget
+info: |
+  ClassTail : ClassHeritage { ClassBody }
+    ...
+    11. Let proto be ObjectCreate(protoParent).
+    ...
+    31. If PrivateBoundIdentifiers of ClassBody contains a Private Name P such that the P's [[Kind]] field is either "method" or "accessor",
+      a. Set F.[[PrivateBrand]] to proto.
+    ...
+
```

Comment:
I think we need to remove this blank line to avoid issues with YAML parser.

Reply:
Thanks for the reminder, but empty lines in between the info block is fine, just avoid leading or trailing lines when possible.